axelrod/strategies/darwin.py

```diff
@@ -23,7 +23,14 @@ class Darwin(Player):
     """
 
     name = "Darwin"
-    memory_depth = float('inf')
+    behaviour = {
```

Comment:
Reposting this from below comment which was in the wrong spot:

What about just changing all of these to be:

manipulates_state
inspects_source
manipulates_source
?

Any thoughts on this @meatballs ?